mathicsscript/fixcairo.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
```

Comment:
This kind of thing is probably better done in setup rather than as a part of the code.